Makefile

```diff
@@ -35,6 +34,9 @@ run-prom-client:
 		-v ./contrib/prometheus/prometheus.yml:/etc/prometheus/prometheus.yml \
 		prom/prometheus
 
+update-local-config:
+	@go run ./config/local/main.go
+
```

Comment:
This feels like something we want to just distribute in our artifacts. I guess this is the same comment as I had below which is I prefer to generate these.

Reply:
Also we should probably have something which ensures that the config files have been regenerated and committed whenever the source is changed, to make sure people don't forget to re-generate.